src/client/utils/dom.js

```diff
@@ -520,8 +520,12 @@ export function isLocation (instance) {
     return instance && isObject(instance) && !isUndefined(instance.href) && !isUndefined(instance.assign);
 }
 
-export function isSVGElement (obj) {
-    return window.SVGElement && obj instanceof window.SVGElement;
+export function isSVGElement (instance) {
+    if (instance instanceof nativeMethods.svgElementClass)
+        return true;
+
+    return instance && isObject(instance) && !isUndefined(instance.ownerSVGElement) && instance.toString &&
```

Comment:
Why we need this `isUndefined` check and this method (and all other methods in `type` utils)?  Why just don't test `instance.ownerSVGElement !== void 0` in place?